pkg/agent/datamodel/helper_test.go

```diff
@@ -24,27 +24,33 @@ func TestValidateDNSPrefix(t *testing.T) {
 		{
 			"empty string",
 			"",
-			errors.New("DNSPrefix '' is invalid. The DNSPrefix must contain between 3 and 45 characters and can contain only letters, numbers, and hyphens.  It must start with a letter and must end with a letter or a number. (length was 0)"),
+			errors.New("DNSPrefix '' is invalid. The DNSPrefix must contain between 3 and 45 characters and can contain only letters," +
```

Comment:
this feels a little silly, we are contorting test code with valid long errors to satisfy linting.

I'm focused less on perfectly following the rules and more catching real issues // consistency.

I guess we already adjusted the line length -- what do you think about this one?

Reply:
I don't mind changing it back and just adding an //nolint here.